src/index.html

```diff
@@ -4,9 +4,414 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "true"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"> 
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#" class="header__home">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__home-link"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a href="#Apple" class="navigation__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a href="#Samsung" class="navigation__link">
+              Samsung
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a href="#Smartphones" class="navigation__link">
+              Smartphones
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#Laptops&Computers"
+              data-qa="nav-hover"
+              class="navigation__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a href="#Gadgets" class="navigation__link">
+              Gadgets
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a href="#Tablets" class="navigation__link">
+              Tablets
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a href="#Photo" class="navigation__link">
+              Photo
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a href="#Video" class="navigation__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <article class="card catalog__item" data-qa="card">
```

Comment:
```suggestion
        <div class="card catalog__item" data-qa="card">
```
product card it's more div than article